lobsterpy/cohp/analyze.py

```diff
@@ -322,6 +333,158 @@ def get_information_all_bonds(self, summed_spins=True):
                     self.seq_labels_cohps.append(type_labels)
                     self.seq_cohps.append(type_cohps)
 
+    def get_site_bond_resolved_labels(self):
+        """
+
+        Returns:
+            dict with bond labels for each site, e.g.
+            {'Na1: Na-Cl': ['21', '23', '24', '27', '28', '30']}
+
+        """
+        bonds = [[] for _ in range(len(self.seq_infos_bonds))]  # type: ignore
+        labels = [[] for _ in range(len(self.seq_infos_bonds))]  # type: ignore
+        for inx, bond_info in enumerate(self.seq_infos_bonds):
+            for ixx, val in enumerate(bond_info.atoms):
+                label_srt = sorted(val.copy())
+                bonds[inx].append(
+                    self.structure.sites[bond_info.central_isites[0]].species_string
+                    + str(bond_info.central_isites[0] + 1)
+                    + ": "
+                    + label_srt[0].strip("0123456789")
+                    + "-"
+                    + label_srt[1].strip("0123456789")
+                )
+                labels[inx].append(bond_info.labels[ixx])
+
+        label_data = {}
+        for indx, atom_pairs in enumerate(bonds):
+            search_items = set(atom_pairs)

```

Comment:
maybe good to think about variable names that are more descriptive?
```suggestion
            searched_atom_pairs = set(atom_pairs)

```